prometheus/go_collector.go

```diff
@@ -236,6 +242,9 @@ func (c *goCollector) Describe(ch chan<- *Desc) {
 func (c *goCollector) Collect(ch chan<- Metric) {
 	c.goroutines.Set(float64(runtime.NumGoroutine()))
 	ch <- c.goroutines
+	n, _ := runtime.ThreadCreateProfile(nil)
+	c.threads.Set(float64(n))
```

Comment:
goroutine already exists. Not sure how much sense it makes to export the others as numbers. Could you make your point what they mean and what they are useful for?